packages/webview_flutter/webview_flutter_wkwebview/pigeons/web_kit.dart

```diff
@@ -364,10 +375,12 @@ abstract class WKWebViewHostApi {
 
   void setNavigationDelegate(int instanceId, int? navigationDelegateInstanceId);
 
+  @ObjCSelector('webViewUrl:')
```

Comment:
This should be `urlForWebView:` (or `urlForWebViewInstance:` or whatever name we decide on for the instance ID parameter).